data-prepper-plugins/translate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/translate/RegexParameterConfiguration.java

```diff
@@ -8,7 +8,7 @@
 
 public class RegexParameterConfiguration {
 
-    private static final boolean DEFAULT_EXACT = true;
+    public final boolean DEFAULT_EXACT = true;
```

Comment:
You can keep this a package private (`final boolean DEFAULT_EXACT = true;`) so that you can reference it in your unit tests.

Reply:
ack.